lib/tessel/deployment/rust.js

```diff
@@ -132,9 +132,10 @@ exportables.remoteRustCompilation = (opts) => {
           try {
             result = JSON.parse(Buffer.concat(buffers));
           } catch (e) {
-            log.error('Invalid JSON response received from cross-compilation server:');
-            log.error('    ', Buffer.concat(buffers).toString());
-            process.exit(1);
+            return reject(tags.stripIndent `
+              Please file an issue on https://github.com/tessel/t2-cli with the following:
+              You received an invalid JSON response from the cross-compilation server.
+                  ${Buffer.concat(buffers).toString()}`);
```

Comment:
This is great feedback for the user but I'm curious if the crash-reporter will also submit an issue if this happens.